core/gather/gatherers/full-page-screenshot.js

```diff
@@ -87,12 +87,12 @@ class FullPageScreenshot extends FRGatherer {
 
     // Height should be as tall as the content.
     // Scale the emulated height to reach the content height.
-    const fullHeight = Math.round(
+    const desiredLayoutViewportHeight = Math.min(metrics.cssContentSize.height, maxTextureSize);
+    const newDeviceHeight = Math.round(
```

Comment:
is the issue that the `min` w/ `maxTextureSize` should happen after this rounded expression?

Reply:
That part of the code might be wrong actually (as in it was correct originally). Looks like the screenshot is sized to `window.outerHeight` not `window.innerHeight` like I thought. Perhaps the only thing that needed to change was the `maxTextureSize - 1`. I'll keep testing this to be sure.